legend-engine-xt-relationalStore-trino-pure/src/main/resources/core_relational_trino/relational/protocols/pure/vX_X_X/extension/metamodel_connection.pure

```diff
@@ -0,0 +1,13 @@
+Class meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::alloy::specification::TrinoDatasourceSpecification extends meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::alloy::specification::DatasourceSpecification
+{
+   host: String[1];
+   port: Integer[1];
+   catalog: String[1];
+   clientTags: String[1];
+   ssl: Boolean[1];
```

Comment:
trustStore properties do not apply when ssl = false right ?

In which case should we consider 

```
ssl : SSLConfiguration[0..1]
..

class SSLConfiguration
{
   trustStorePathRef : String[1]
   trustStorePasswordRef : String[1];
}
```

Reply:
Let's discuss.